app/views/registration/early_years_experiences/edit.html.slim

```diff
@@ -0,0 +1,18 @@
+= render 'user/debug'
+
+- content_for :page_title do
+  = html_title t(:heading, scope: :early_years_experience)
+
+= govuk_back_link href: edit_registration_role_type_path
+
+.govuk-grid-row
+  .govuk-grid-column-full
+    = form_for form, url: registration_early_years_experience_path, method: :patch do |f|
+      = f.govuk_error_summary
+
+      = f.govuk_fieldset legend: { text: t('register_early_years_experience.heading')  }, aria: { required: true } do
+        .data-hj-suppress
+          div class='govuk-!-margin-top-4'
+            = f.govuk_collection_radio_buttons :early_years_experience, form.early_years_experiences, :name, :name, legend: { text: t('early_years_experienc.heading'), class: 'govuk-visually-hidden govuk-!-padding-top-9' }, form_group: { classes: %w[data-hj-suppress] }
+
+      = f.govuk_submit t('links.continue')
```

Comment:
The code changes look good 👍  I have checked this locally. One very small request: please could you add a new line in the files such as this that have been highlighted like this. This is also for `data/early-years-experience.yml`

Reply:
[73b1a5b](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/1048/commits/73b1a5b456f09f3b1cee1b4afe0e8287d4f7a418)